HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java

```diff
@@ -29,25 +32,58 @@
 @Controller
 public class RecruiterController {
     @Autowired
-    private RecruiterService recruiterService;
+    private RecruiterService recruiterService;    
+
+    @Autowired
+    private ApplicantService applicantService;
 
     @RequestMapping("/")
     public String homePage(Model model) {
+    	model.addAttribute("title", "Home");
+        return "index";
+    }
+    
+    @RequestMapping("/index")
+    public String indexPage(Model model) {
+    	model.addAttribute("title", "Home");
+        return "index";
+    }
+    
+    @RequestMapping("/recruiter")
```

Comment:
Why you have used RequestMapping for every methods? Why not used methods like GetMapping, PostMapping and all

Reply:
I am still reading about all the annotations and the advantages of one over another. That is why i was using RequestMapping as a general tag